src/app.js

```diff
@@ -1 +1,203 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+console.log('Hallo daar!');
+const ulElement = document.getElementById('countries');
+
+async function getAll() {
+    try {
+        const response = await axios.get('https://restcountries.com/v2/all');
+        // console.log(response.data);
+        return response.data;
+    } catch (e) {
+        console.error(e);
+    }
+}
+
+function doListOfCountries(){
+    getAll()
+        .then((result) => {
+            sortLandsByPopulation(result);
+            result.map((land) => {
+                const listElement = document.createElement('li');
+                const spanElement = document.createElement('span');
+                spanElement.setAttribute('class', 'flag-land-name');
+
+                const flagElement = document.createElement('img');
+                flagElement.src = land.flags.png;
+                flagElement.alt = `Flag of ${land.name}`;
+                flagElement.setAttribute('class', 'flag');
+                spanElement.appendChild(flagElement);
+
+                const landNameElement =document.createElement('h3');
+                landNameElement.textContent = land.name;
+                landNameElement.setAttribute('class', regioColor(land.region));
+                spanElement.appendChild(landNameElement);
+                listElement.appendChild(spanElement);
+
+                const population = document.createElement('p');
+                population.textContent = `Has a population of ${land.population} people`;
+                listElement.setAttribute('class', `list`)
+                listElement.appendChild(population);
+                ulElement.appendChild(listElement);
+            });
+        })
+        .catch(() => {
+            console.error('Not found');
+        });
+}
+
+function sortLandsByPopulation(listOfLands) {
+    listOfLands.sort((a, b) => {
+        if (a.population > b.population) {
+            return 1;
+        }
+        if (a.population< b.population) {
+            return -1
+        }
+        return 0;
+    });
```

Comment:
Dit kun je nog iets korter schrijven
```suggestion
    listOfLands.sort((a, b) => a.population - b.population);
```